src/nl/rubensten/texifyidea/navigation/NavigationItemUtil.kt

```diff
@@ -17,18 +19,25 @@ object NavigationItemUtil {
 
     @JvmStatic
     fun createLabelNavigationItem(psiElement: PsiElement): NavigationItem? {
-        when (psiElement) {
-            is LatexCommands -> return GoToSymbolProvider.BaseNavigationItem(psiElement,
-                    psiElement.requiredParameter(0) ?: return null,
-                    if (psiElement.name in Magic.Command.labels) TexifyIcons.DOT_LABEL else TexifyIcons.DOT_BIB
-            )
-            is BibtexId -> return GoToSymbolProvider.BaseNavigationItem(psiElement,
+        return when (psiElement) {
+            is LatexCommands -> {
+                val text = psiElement.extractLabelName()
+                if (text == "")  {
+                    return null
+                }
+                return GoToSymbolProvider.BaseNavigationItem(psiElement,
+                        text,
+                        if (psiElement.name in Magic.Command.labels ||
+                                TexifySettings.getInstance().labelCommands.containsKey(psiElement.name))
+                            TexifyIcons.DOT_LABEL else TexifyIcons.DOT_BIB
```

Comment:
Add braces to make it better readable.

```kotlin
                        if (psiElement.name in Magic.Command.labels ||
                                TexifySettings.getInstance().labelCommands.containsKey(psiElement.name)) {
                            TexifyIcons.DOT_LABEL
                        }
                        else TexifyIcons.DOT_BIB
```

Reply:
Okay, I'm a little bit confused.
```kotlin
if (condition) {
    true
}
else false
```
If I get you right, the `true`-part should be in braces, the `false`-part could be written without braces in case there is only one statement? Because here you added the braces and in this other [comment](https://github.com/Ruben-Sten/TeXiFy-IDEA/pull/815#discussion_r275389152) you removed the braces.

Reply:
I do that with the `else` clause, when the return value of an if/else statement. It removes a slight bit of clutter in my opinion.

```kotlin
val oneLiner = if (true) 1 else 0

val assignment = if (true) {
    more.complex.stuff().toIntOrNull()
}
else null

if (true) {
    action1()
    action2()
}
else {
    action3()
}
```

Reply:
Oh, okay, now I understood your schematics. Then I will do it this way.
But I'm a little bit confused again... In this case it was like your `oneLiner` but with a big condition. Should I then use braces true?